### DISCUSSION TITLE
Great Supporting Evidence

### DISCUSSION THREAD

**Discussion by BrewingGeek** [jasonacox/Powerwall-Dashboard#275] [General]
Just wanted to share this to all and a special thanks to all the people that spent time making pypowerwall what it is. 

I had my system turned on 09/01/2022, but I was collecting data before that date, nice little baseline.  The day that the system was turn on, I had an grid event, grid went down. The system failed.. Luckily the outage was only a few minutes.  The installer blew me off about the system not working, not the first time the ignored me. A few months later, November, I observed with the collected data that the system was acting differently.  I got on the phone with my installer and finally had them contact Tesla.  And within 30 seconds of Tesla looking at the system they stated that the system was installed/configured wrong.  Thanks to the data collected I was able to acknowledge see they same error and it backed up my claim that the system was not operating properly.  A similar situation happened in April 2023. Here the tech arrived and the data collected again proved a system issue. One highlight was the tech stating how shook he was that the system data being collected and displayed by me was live..

**Comment by anderep** [comment]
So, did they fix the configureation?

**Reply by BrewingGeek** [reply]
Yes so far all accounts..  They had a CT configured wrong  so it  appeared that the house was generating electricity along with solar..  And the last time I was told by the tech that there is 'at times' compatibility issue between the SolarEdge inverter and the Tesla Powerwall 2. This issue took SolarEdge support to fix. That issue was found we the tech and I exercised a mock grid down (turn the main breakers of to the house). The unit did not provide solar to the house ) s full sunny day).

It does rub me the wrong way that the installer does NOT run and test on the system to ensure proper operation.  All they seem to care about is that it produces solar electricity. Nor do they execute any kind of walk through after the installation.  This is important in a few ways, one when you need to do a system reset. As the state there is sequence of switched and breakers that need to be turned off and back on. And the items are not identified or referenced clearly in the instruction they provide.

  This is the plot that the system was NOT configured properly and the House was generating electricity along with solar
![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/48930037/406d22e3-9daf-4fbf-bad4-fcf32ef04395)

**Reply by jasonacox** [reply]
Thanks for sharing your story @BrewingGeek !  Is that dashboard something you built using data from pypowerwall?

**Reply by anderep** [reply]
Yep home shouldn’t go negative. Put perhaps you missed your opportunity to showcase how you created free power on YouTube. 
Tesla installed my system, the solar CT on the gateway read about 10% high causing my calculated house load to increase. They came back and installed a NEURIO for my solar, one CT for tesla and one for enphase.

**Reply by BrewingGeek** [reply]
wished I thought of the YouTube idea back then.

**Comment by BrewingGeek** [comment]
That is my own dashboard..
